### PR TITLE
Fix response example to show an array of objects is returned [SA-15481] develop

### PR DESCRIPTION
**Pull request by tomcdonnell**
### Before change:
image

### After change:
image

### Actual response:
image

### Documentation (already correct, no changes needed):
image

### Actual response:
image

### CODE REVIEW

File: openapi/components/responses/userGroupMembership-list.yaml [hunk 1]
```diff
@@ -7,6 +7,6 @@ content:
         A group within Schoolbox.
         
         This contains some of the folder's fields in this context.
-      type: object
-      properties:
+      type: array
```
Comment (armatronic) [inline openapi/components/responses/userGroupMembership-list.yaml:10]: This raises an error when I attempt to build it:

```
npm run build --build-version=23.0.0-alpha1
```

raises:
```
[1] openapi/components/schemas/userGroupMembership-readableFields.yaml:1:1 at #/id
...
```

I think that this is supposed to be something like:
```
type: object
items:
  $ref: ../schemas/userGroupMembership-readableFields.yaml
```

but even that still shows something awkward for the response for the "get a user" endpoint:
image

**Review by armatronic — CHANGES_REQUESTED** [review]

File: openapi/components/responses/userGroupMembership-list.yaml (hunk 1 above)
Comment (tomcdonnell) [reply openapi/components/responses/userGroupMembership-list.yaml:10]: I'm confused @armatronic.  I don't get the error when I build, and in the documentation for 'get a user', I see a different description for 'groups', that does not mention 'indexed by folder id of the groups'.
image
image

What are you and I doing differently?

Comment (tomcdonnell) [reply openapi/components/responses/userGroupMembership-list.yaml:10]: Also I noticed that I can't run 'gitsync develop' when using schoolbox-api-docs.

It tells me that my branch is up to date with origin/develop, but also that 'upstream' does not appear to be a git repository.

I tried fixing with `git branch --set-upstream-to...`, but that has not worked.

Any ideas?
image

Comment (tomcdonnell) [reply openapi/components/responses/userGroupMembership-list.yaml:10]: I manually edited my .git/config file to the following, and then I was able to run `git pull`, which updated many (88) files.  Once I had done that, I tried to rebuild and now I get the error you listed above, plus eight other errors all concerning userGroupMembership files.
image

So I think I am now seeing what you are seeing.  I also think I might be better off starting this PR from scratch now that my local branch is up to date.  I will try re-branching from develop and starting again.

Comment (tomcdonnell) [reply openapi/components/responses/userGroupMembership-list.yaml:10]: One more update for my future self.

I deleted my local hotfix-develop/SA-15481 branch, and added a [remote "upstream"] block (to match my working configuration for the schoolbox repository.  Now `gitsync develop` works, and I'm confident all is as it should be.
image

**Comment by tomcdonnell** [comment]
This was all messed up due to my local git develop branch not being up to date.  I will submit a new PR.

File: openapi/components/responses/userGroupMembership-list.yaml (hunk 1 above)
Comment (armatronic) [reply openapi/components/responses/userGroupMembership-list.yaml:10]: Sorry I didn't notice this earlier, but it looks like you've found yourself in the right place.

**Pull request closed without merging**